apps/web/core/components/features/task/task-status.tsx

```
@@ -1221,113 +1207,105 @@ export function MultipleStatusDropdown<T extends TStatusItem>({
-						enterTo="transform scale-100 opacity-100"
-						leave="transition duration-75 ease-out"
-						leaveFrom="transform scale-100 opacity-100"
-						leaveTo="transform scale-95 opacity-0"
-						as="div"
-						className={clsxm(
-							'absolute right-0 left-0 z-[999] min-w-min outline-none',
-							issueType === 'issue' && ['left-auto right-auto'],
-							dropdownContentClassName
-						)}
+
+							onRemoveSelected && onRemoveSelected();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)